text/stacks/0004-jammy-jellyfish.md

```diff
@@ -0,0 +1,113 @@
+# Stacks based on Ubuntu 2022.04: Jammy Jellyfish
+
+## Summary
+
+A set of stacks based on the Ubuntu 2022.04 LTS (Jammy Jellyfish) release base
+image should be developed, released, and maintained by the Stacks subteam. Like
+the existing Bionic stacks, these new stacks should come in "full", "base", and
+"tiny" variants with similar, if not identical, sets of packages pre-installed.
+
+## Motivation
+
+Ubuntu provides a long-term-support (LTS) release every 2 years in April. These
+releases are supported by Canonical for 5 years. The current Bionic base image,
+first released in April 2018, will therefore be going out of support at the end
+of March 2023. In order to ensure that Paketo continues to provide a supported
+Ubuntu-based stack for our users, we should produce a new set of stack images,
+based on Jammy, alongside the current Bionic stacks.
+
+## Detailed Explanation
+
+### Variants
+
+The Jammy stack will be delivered in 3 variants that align to those already
+offered in the Bionic stack. The variants will be called `base`, `full`, and
+`tiny` just as they are in the Bionic stack. Each should be developed to offer
+roughly the same set of OS-level package support as their Bionic equivalent.
+
+### Stack IDs
+
+Each of these stacks will declare their stack ID as `io.paketo.stack.jammy`.
+This choice differs from Bionic in that we will **NOT** differentiate between
+the `tiny` and `base/full` variants using the stack ID.
```

Comment:
I think there are some existing cases where paketo buildpacks use the stack ID to differentiate between tiny and base in order to behave differently. One example is the Procfile buildpack which behaves differently in the presence and absence of a shell.

I suspect most of these cases are about the presence of bash so maybe we can just remove these as we try to encourage shell-less images? But I think there is at least one example with java native images where the buildpack needs to know whether to statically compile in certain libraries (cc @dmikusa-pivotal )

Reply:
I have this concern as well. Raised above.  https://github.com/paketo-buildpacks/rfcs/pull/172#discussion_r847604706

Reply:
Resolved in https://github.com/paketo-buildpacks/rfcs/pull/172#discussion_r853193113